src/protocol/protocol.c

```diff
@@ -131,6 +130,9 @@ void PROTOCOL_Load(int no_dlg)
 {
     (void)no_dlg;
 #ifdef ENABLE_MODULAR
+    FATFS ModuleFAT;
+    FILE *fh;
+
     if(! PROTOCOL_HasModule(Model.protocol)) {
```

Comment:
This does add to the stack usage, right (compared to reusing the font struct).  It is probably not relevant, but do you have an idea of how our worst-case stack usage looks?

Reply:
Understand you concern. FAT structure for devo7e is pretty small 0x28 bytes. Our major stack consumer is still in mixer code. Check the issue I reported before #479. Since we have any callback, the script I use cannot really sum up the stack usage. But the report still gives u a bit idea on how much stack we are using.

Reply:
Just checking.  I have no real concern about it

Reply:
please merge. then I can start gcc8.2 testing.